tests/app/shared/issue/duplicatedIssues/duplicated-issues.component.spec.ts

```diff
@@ -0,0 +1,147 @@
+    const secondDummyIssue = Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION_LOW_SEVERITY_DOCUMENTATION_BUG, dummyTeam);
+
+    const permissionService: any = jasmine.createSpyObj('PermissionService', ['isTeamResponseEditable', 'isTutorResponseEditable']);
+    const issueService: any = jasmine.createSpyObj('IssueService', ['getDuplicateIssuesFor', 'updateIssueWithComment', 'updateIssue', 'updateLocalStore']);
+    const phaseService: any = jasmine.createSpyObj('PhaseService', [], { currentPhase: Phase.phaseModeration });
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                DuplicatedIssuesComponent
+            ],
+            providers: [
+                IssueService, ErrorHandlingService, PhaseService, PermissionService, {
+                    provide: HAMMER_LOADER,
+                    useValue: () => new Promise(() => {})
+                  }
+            ],
+            imports: [
+                MaterialModule, RouterTestingModule, BrowserDynamicTestingModule, ApolloTestingModule, HttpClientTestingModule
+            ]
+        })
+        .overrideProvider(PermissionService, { useValue: permissionService })
+        .overrideProvider(IssueService, { useValue: issueService})
+        .overrideProvider(PhaseService, { useValue: phaseService})
+        .compileComponents();
+    }));
+
+    beforeEach(() => {
+        dummyDuplicatedIssues = of([firstDummyIssue, secondDummyIssue]);
+        issueService.getDuplicateIssuesFor.and.callFake(() => dummyDuplicatedIssues);
+
+        thisIssue = Issue.createPhaseModerationIssue(ISSUE_PENDING_MODERATION, dummyTeam);
+        firstDummyIssue.duplicateOf = thisIssue.id;
+        secondDummyIssue.duplicateOf = thisIssue.id;
+
+        fixture = TestBed.createComponent(DuplicatedIssuesComponent);
+        component = fixture.componentInstance;
+        component.issue = thisIssue;
+        fixture.detectChanges();
+
+        debugElement = fixture.debugElement;
+        nativeElement = fixture.nativeElement;
+    });
+
+    it('should create a chip for each duplicated issue', () => {
+        // Check that the number of chips is the same as the number of duplicate issues
+        const matChipListLength: number = debugElement.queryAll(By.css('.mat-chip')).length;
+        const duplicateIssueslength: number = getDuplicateIssuesLength();
+        expect(matChipListLength).toEqual(duplicateIssueslength);
+
+        // Check that first chip contains information on first issue in the duplicate issues array
+        const matChipAnchor: HTMLElement = nativeElement.querySelector('a');
+        expect(matChipAnchor.innerText).toEqual(`#${firstDummyIssue.id}`);
+    });
+
+    it('should only allow cancellation of duplicate status if team/tutor response is editable', () => {
+        // Team/tutor response is not editable
+        permissionService.isTeamResponseEditable.and.callFake(() => false);
```

Comment:
Maybe we can abstract these lines into a helper method since they are repeated throughout this set of test cases as well! 😄

Reply:
I didn't want to abstract those lines out cause they are a one-liner, but thinking back, I think it will be easier to understand if I abstract it out! Thanks for pointing it out:)

Reply:
Aights!